packages/react-components/react-color-picker-preview/library/docs/Spec.md

```diff
@@ -2,62 +2,458 @@
+  - [ColorSlider](https://react-spectrum.adobe.com/react-spectrum/ColorSlider.html)
+
+### Components
+
+| Purpose                                                                  | Fabric (V8)    | V9          | Matching? |
+| ------------------------------------------------------------------------ | -------------- | ----------- | --------- |
+| Component responsible for color editing using ColorArea and ColorSliders | ColorPicker    | ColorPicker | ⚠️        |
+| ColorArea allows user to pick a color using two channels                 | ColorRectangle | ColorArea   | ⚠️        |
+| ColorSlider allows user to pick a color using individual channel         | ColorSlider    | ColorSlider | ⚠️        |
+| AlphaSlider allows user to pick a color using alpha channel              |                | AlphaSlider | ❌        |
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+```jsx
+import {
+  ColorPicker,
+  ColorArea,
+  ColorSliderProps,
+  AlphaSlider,
+  HueSlider,
+  ColorPickerOnSelectEventHandler,
+} from '@fluentui/react-color-picker-preview';
+
+export const Default = () => {
+  const COLOR = 'rgba(0, 255, 170, 1)';
+  const [selectedColor, setSelectedColor] = React.useState(COLOR);
+  const handleChange: ColorPickerOnSelectEventHandler = (_, data) => {
+    setSelectedColor(data.selectedColorHex);
+  };
+
+  return (
+    <>
+      <ColorPicker color={selectedColor} onChange={handleChange}>
+        <ColorArea />
+        <AlphaSlider />
+        <HueSlider />
+      </ColorPicker>
+      <div style={{ backgroundColor: `${selectedColor}` }} />
+    </>
+  );
+};
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+### Slider variants
+
+- horizontal (default)
+- vertical
+
+ColorSliders might represent different color channels.
+
+### Shapes
+
+- `square` (default)
+- `rounded`
+
+### Size
+
+Size will be the same as in Office products.
+For custom sizes users might want to customize it via CSS.
+
+### States
+
+- `rest`
+- `focused`
+
+### Color support
+
+- RGB(A)
+- Hex
+- HSL(A)
+- HSV/HSB
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+### ColorPicker
+
+| Property   | Values              | Default   | Purpose                                |
+| ---------- | ------------------- | --------- | -------------------------------------- |
+| color      | `string`            |           | Sets color value                       |
+| onChange   | `function`          | undefined | Callback called when color is selected |
+| shape      | `square`, `rounded` | `square`  | Sets shape                             |
+| step       | `number`            | 1         | Step for arrow navigation              |
+| customStep | `number`            | 3         | Fast navigation                        |
```

Comment:
<img width="552" alt="image" src="https://github.com/user-attachments/assets/65beb627-b596-4251-8a25-422e630f07bf">

What is "Fast navigation"?

Reply:
It's when step is bigger then 1, by default it's 3. So a user can navigate faster on a slider.